pkg/storage/tableland/common/sqlfactory.go

```diff
@@ -38,45 +39,23 @@ func (s *SQLFactory) SQLForInsertingParts(table string, parts []local.Part) (str
 }
 
 // SQLForInsertingLayers returns the SQL statement.
-func (s *SQLFactory) SQLForInsertingLayers(table string, cid string, layers []local.Layer) (string, error) {
+func (s *SQLFactory) SQLForInsertingLayers(table string, layers []local.Layer) (string, error) {
```

Comment:
Changes in here just to account for the fact that imagery root cids are now stored in a lookups table.